### ISSUE TITLE
Default Learning Rate in pretraining/args.py

### ISSUE THREAD

**Issue by bobfromjapan** [hirokatsukataoka16/FractalDB#7]
I am conducting a replication experiment to create a FractalDB pretrained VGG16 with TensorFlow, referencing this repository. 
I have a concern regarding a hyperparameter: the default value for `learning_rate` in `pretraining/args.py` is set to `0.1`, but shouldn't it be `0.01`? 
I checked the paper, and it mentions that the initial learning rate is 0.01. Just to be sure, I tried training for about 30 epochs with a learning rate of 0.1, but it was too high and the loss did not decrease at all.

https://github.com/hirokatsukataoka16/FractalDB-Pretrained-ResNet-PyTorch/blob/1b8fb9fe7a6f496d4a2acc4ae5e1c1ab1eab0608/pretraining/args.py#L20C5-L20C89

Thanks in advance.

**Comment by hirokatsukataoka16** [comment]
Please try to learn a network with learning rate 0.01. However, VGG architectures have not been optimized in our experiments. We also mentioned the published journal as follows.

> However, during the experiment, we could not optimize FractalDB pre-trained VGGNet-{16, 19}. Therefore, accuracies with VGGNet-{16, 19} are not included in the table.
https://link.springer.com/content/pdf/10.1007/s11263-021-01555-8.pdf

**Comment by bobfromjapan** [comment]
Thanks for your reply!

Indeed, simply setting the learning rate to 0.01 did not result in the convergence.
Regardless of whether this method is correct, it was confirmed that reducing the Dropout ratio from 0.8 to 0.2 makes progress in training.

I have uploaded the FractalDB-1K pretrained VGG16 weights for TensorFlow I created to the following link.
I note that I use this repository as a reference and inherite the MIT license, but I will withdraw it if it is not preferred, so please let me know.

https://huggingface.co/bobfromjapan/tf_vgg16_fractaldb_pretrained
